test/routes/confirmRules.route.test.js

```diff
@@ -52,7 +53,7 @@ lab.experiment('Confirm that your operation meets the rules page tests:', () =>
 
       const parser = new DOMParser()
       const doc = parser.parseFromString(res.payload, 'text/html')
-      Code.expect(doc.getElementById('confirm-rules-heading')).to.exist()
+      Code.expect(doc.getElementById('confirm-rules-heading').firstChild.nodeValue).to.equal(Constants.Routes.CONFIRM_RULES.pageHeading)
```

Comment:
We shouldn't be referencing values in the constants from the tests because if the constant is/becomes incorrect then the test will still pass. 

Instead we should be testing that it is equal to the string 'Confirm your operation meets the rules' rather than that the value that is held in the Constants file.